packages/modelviewer.dev/examples/loading/index.html

```diff
@@ -330,7 +330,50 @@ <h4>
         </div>
       </div>
     </div>
-  
+
+    <div class="sample">
+      <div id="meshoptSupport" class="demo"></div>
+      <div class="content">
+        <div class="wrapper">
+          <div class="heading">
+            <h2 class="demo-title">Meshopt support</h2>
+            <h4>
+              &lt;model-viewer&gt; supports loading glTF models that
+
+              use <a href="https://github.com/KhronosGroup/glTF/blob/master/extensions/2.0/Vendor/EXT_meshopt_compression/README.md" target="_blank" rel="noopener">the Meshopt compression extension</a>.
+            </h4>
+            <h4>
+              In order to load such models, an auxiliary decoder is
+              required, and is not enabled by default.
+            </h4>
+          </div>
+          <example-snippet stamp-to="meshoptSupport" highlight-as="html">
+            <template>
+<model-viewer camera-controls alt="A 3D model of a mechanical coffee mug contraption" src="../../shared-assets/models/coffeemat.glb">
+</model-viewer>
+            </template>
+          </example-snippet>
+          <p>
+            Enable support for Meshopt compression by providing
+            <code>meshoptDecoderLocation</code>:
+          </p>
+          <example-snippet inert-script highlight-as="html">
+            <template>
+<script>
+ModelViewerElement = self.ModelViewerElement || {};
+ModelViewerElement.meshoptDecoderLocation = 'https://unpkg.com/meshoptimizer@0.16.0/meshopt_decoder.js';
```

Comment:
Can  we switch these  (and the previous example) from `ModelViewerElement` to `self.ModelViewerElement`? I've been meaning to do this anyway, since as written it doesn't compile as a module script.